src/RippleFeedback/RippleFeedback.react.js

```diff
@@ -23,7 +22,7 @@ const defaultProps = {
 };
 
 function isCompatible() {
-    if (Platform.OS === 'ios') {
+    if (Platform.OS === 'ios' || Platform.OS === 'web') {
```

Comment:
Is there something like Platform.OS === 'web'? it'll be always false, right?

Reply:
react-native-web uses the 'web' platform:
https://github.com/necolas/react-native-web/blob/master/src/apis/Platform/index.js

Reply:
Nice, sorry, did not know that ... Thanks